apis/v1alpha1/httproute_types.go

```diff
@@ -23,6 +23,10 @@ import (
 type HTTPRouteSpec struct {
 	// Hosts is a list of Host definitions.
 	Hosts []HTTPRouteHost `json:"hosts,omitempty"`
+
+	// Gateways defines which Gateways can use this Route.
+	// +kubebuilder:default={allow: "SameNamespace"}
+	Gateways *RouteGateways `json:"gateways,omitempty"`
```

Comment:
This field is not a pointer in other routes but is a pointer here.
Most likely an oversight, if not why pointer?

Reply:
Good catch, I think this was a leftover from when this was an optional field.